serve/mlc_serve/engine/engine_common.py

```diff
@@ -132,6 +133,31 @@ def detokenize_incrementally(
     return delta
 
 
+def logprob_detokenize(tokenizer: TokenizerP, logprob_info: Optional[LOGPROBS_TYPE]) -> Optional[LOGPROBS_TYPE]:
+    """Detokenize logprob information"""
+    if logprob_info is None:
+        return None
+    (res, res_logprob), top_tokens = logprob_info
+    top_tokens = list(top_tokens)
```

Comment:
Is this needed?

Reply:
This is becaused some previous implementation will return ziped results as a list of tuples. Will fix.